cogs/roles.py

```diff
@@ -257,6 +257,23 @@ async def create_role(self, ctx, *, role: Optional[str] = None):
         await ctx.guild.create_role(name=role, reason="Created with Canary")
         await ctx.send("Role created successfully.")
 
+    @commands.command()
+    async def inchannel(self, ctx):
+        """Returns list of users in current channel"""
+        channel = ctx.message.channel
+        members = channel.members
+
+        channel_users = list(map(lambda m: str(m) + "\n", members))
+        header = f"List of users in {channel.mention} - {len(members)}"
```

Comment:
Since this is used into an embed title, "channel.mention" doesn't work there, instead you get `<#channel_id>`. Use `#{channel}` instead

Reply:
Resolved in fc6d147